l10n_br_coa/models/template_br_oca.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2020 KMEE
+# Copyright 2025 Escodoo - Marcel Savegnago <marcel.savegnago@escodoo.com.br>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
```

Comment:
@marcelsavegnago vc mudou o nome do arquivo... Se isso for realmente necessário é algo que devemos fazer na 16.0 já para facilitar de sincronizar as branches depois...